redbeams/src/main/resources/schema/app/20220927124116_CB-18629_-_All_flow_logs_should_have_non_zero_resource_id.sql

```diff
@@ -0,0 +1,17 @@
+-- // CB-18629 - All flow logs should have non zero resource id
+-- Migration SQL that makes the change goes here.
+UPDATE flowlog f
+SET resourceid =
+    (
+    SELECT MAX(resourceid)
+    FROM flowlog
+    WHERE f.flowid = flowid
+    )
+WHERE resourceid = 0;
```

Comment:
do we really need this for finalized flows?

Reply:
I have not tested filtering for non-finalized but finalized get purged periodically so it should not matter.